.ci/workspace-startup/che6_workspace.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
could we drop che 6 at all ?

Reply:
of course, will update this to drop the che6. it's not being used by the scripts anyways.

Reply:
deleted.